### PR TITLE
Mysql kustomize v3 manifest for both generic and gcp stack

### PR DESCRIPTION
**Pull request by Bobgy**
This is based on https://github.com/kubeflow/manifests/pull/1243, please merge it first.
This PR can be reviewed with https://github.com/kubeflow/manifests/pull/1247/files/6151fe9f4a0c48a8a256227232b3a31be7cca831..2544716e74fd18a27bb41b272f65546497c5fbf9 now.

**Which issue is resolved by this Pull Request:**
Resolves https://github.com/kubeflow/pipelines/issues/3975

**Description of your changes:**
Mysql kustomize v3 manifest for both generic and gcp stack

**Checklist:**
- [x] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate-changed-only`
    3. `make test`

### CODE REVIEW

File: hack/pull_kfp_upstream.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+export PIPELINES_SRC_REPO=https://github.com/kubeflow/pipelines.git
+export PIPELINES_VERSION=1.0.0-rc.1
```
Comment (rmgogogo) [inline hack/pull_kfp_upstream.sh:4]: in the future for new version updates, we just 

1. update this value

2. run this shell?

Comment (Bobgy) [reply hack/pull_kfp_upstream.sh:4]: Yes, exactly

**Pull request merged**